media-libs/devil/devil-1.8.0.recipe

```diff
@@ -54,20 +67,33 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	runConfigure ./configure CFLAGS=-Dpng_set_gray_1_2_4_to_8=png_set_expand_gray_1_2_4_to_8
+	cd DevIL
+	mkdir -p haiku-build
+	cd haiku-build
+
+	cmake .. \
+		-DCMAKE_INSTALL_PREFIX:PATH=$prefix \
+		-DCMAKE_CXX_FLAGS="-fpermissive"
```

Comment:
Does this project use GNUInstallDirs in their CMake files? If so, you can use $cmakeDirArgs and probably ditch the directory moves below.

If they still have the configure script, maybe stick with that...